backends/dnf/notify_packagekit.cpp

```diff
@@ -0,0 +1,80 @@
+#include <libdnf5/base/base.hpp>
+#include <libdnf5/common/exception.hpp>
+
+#include <algorithm>
+#include <cstdlib>
+
+using namespace libdnf5;
+
+namespace {
+
+constexpr const char * PLUGIN_NAME{"template"};
+
+constexpr plugin::Version PLUGIN_VERSION{.major = 1, .minor = 0, .micro = 0};
+
+constexpr const char * attrs[]{"author.name", "author.email", "description", nullptr};
+constexpr const char * attrs_value[]{"Alessandro Astone", "ales.astone@gmail.com",
+        "Notify packagekitd when packages are installed, updated, or removed."};
+
+class NotifyPackagekitPlugin : public plugin::IPlugin {
+public:
+    NotifyPackagekitPlugin(libdnf5::Base & base, libdnf5::ConfigParser &) : IPlugin(base) {}
+
+    const char * get_name() const noexcept override { return PLUGIN_NAME; }
+
+    plugin::Version get_version() const noexcept override { return PLUGIN_VERSION; }
+
+    PluginAPIVersion get_api_version() const noexcept override { return PLUGIN_API_VERSION; }
+
+    /// Add custom attributes, such as information about yourself and a description of the plugin.
+    /// These can be used to query plugin-specific data through the API.
+    /// Optional to override.
+    const char * const * get_attributes() const noexcept override { return attrs; }
+    const char * get_attribute(const char * attribute) const noexcept override {
+        for (size_t i = 0; attrs[i]; ++i) {
+            if (std::strcmp(attribute, attrs[i]) == 0) {
+                return attrs_value[i];
+            }
+        }
+        return nullptr;
+    }
+
+    void post_transaction(const libdnf5::base::Transaction & transaction) override;
+};
+
+void NotifyPackagekitPlugin::post_transaction(const libdnf5::base::Transaction & transaction) {
+    std::system("gdbus call --system --dest org.freedesktop.PackageKit --object-path /org/freedesktop/PackageKit --method org.freedesktop.PackageKit.StateHasChanged posttrans");
+}
```

Comment:
Can you use `sdbus-cpp` here instead of shelling out to `gdbus`?

Reply:
I'll see if it isn't necessarily complex.
Otherwise `gdbus` or alternatively `busctl` are pretty much guaranteed to be installed already so we wouldn't be adding new dependencies.